python/ray/data/datasource/mongo_datasink.py

```diff
@@ -1,12 +1,14 @@
 import logging
```

Comment:
For consistency with the `Datasource` implementations, could we move the `Datasink` implementations to `_internal` and also remove leading underscores from the names (e.g., move `ray.data.datasource.mongo_datasink._MongoDatasink` to `ray.data._internal.datasource.mongo_datasink.MongoDatasink`?

Reply:
will do

Reply:
https://github.com/ray-project/ray/pull/46331